specs/pkg-js/trustless-pathing/unixfs_20m_variety.js

```diff
@@ -0,0 +1,30 @@
+import { readFile } from 'node:fs/promises'
+import { parseCase } from './testcase.js'
+import { parse, index, patch, toString } from 'testmark.js'
+
+const file = '../../transport/trustless-pathing/fixtures/unixfs_20m_variety.'
+
+function pathToFixture(typ) {
+  return new URL(file + typ, import.meta.url)
+}
+
+export function unixfs20mVarietyCar() {
+  return pathToFixture('car')
+}
+
+export async function unixfs20mVarietyCases() {
+  const file = pathToFixture('md')
+  const contents = await readFile(file, 'utf-8')
```

Comment:
Synchronous allows consumers to call and iterate test cases to define individual tests at the top level.

```suggestion
export function unixfs20mVarietyCases() {
  const file = pathToFixture('md')
  const contents = readFileSync(file, 'utf-8')
```